instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/RuntimeMetricsBuilder.java

```diff
@@ -81,13 +82,20 @@ public RuntimeMetricsBuilder enableExperimentalJmxTelemetry() {
     return this;
   }
 
+  /** Enable capture GC cause. */
```

Comment:
Use `Enable the capture of the jvm.gc.cause attribute within the jvm.gc.duration metric.` as the comment.